recipes/pydash/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pydash" %}
+{% set version = "4.2.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "58573ec724d696a2268fec6bcb4e19549cb15876ec03837c4455115f95712e6a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - pydash
+
+about:
+  home: https://github.com/dgilland/pydash
+  license: MIT
+  license_file: LICENSE.rst
+  summary: The kitchen sink of Python utility libraries for doing "stuff" in a functional way. 
+  description: "The kitchen sink of Python utility libraries for doing stuff in a functional way. Based on Lo-Dash JS library"
+  doc_url: https://pydash.readthedocs.io/en/latest/
+  dev_url: https://github.com/dgilland/pydash
+
+extra:
+  recipe-maintainers:
+    - souravsingh
+    - dgilland
```

Comment:
@dgilland -- are you ok being added as a co-maintainer of the `pydash` package on conda-forge?

Reply:
What does that involve? I'm not familiar with what responsibilities a maintainer/co-maintainer has with conda-forge.

Reply:
That seems fine by me although my bandwidth is fairly low these days. But having @souravsingh as the primary would be best.

Reply:
I can take up the full responsibility of maintaining the feedstock.